### PR TITLE
Proxy more display methods

### PR DESCRIPTION
**Pull request by sivel**
##### SUMMARY

Proxy more display methods

##### ISSUE TYPE

- Bugfix Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/utils/display.py [hunk 1]
```diff
@@ -425,31 +425,39 @@ def display(
...
+    @proxy_display
     def vvvvv(self, msg: str, host: str | None = None) -> None:
         return self.verbose(msg, host=host, caplevel=4)
 
+    @proxy_display
     def vvvvvv(self, msg: str, host: str | None = None) -> None:
         return self.verbose(msg, host=host, caplevel=5)
 
+    @proxy_display
     def debug(self, msg: str, host: str | None = None) -> None:
```
Comment (bcoca) [inline lib/ansible/utils/display.py]: i would not proxy the verbose/v as they all call display in the end, which is already proxied, this work can still be done in the fork w/o impacting output

Comment (sivel) [reply lib/ansible/utils/display.py]: This is more for book keeping for runner, runner needs to know exactly what method was called.

**Comment by sivel** [comment]
After looking more closely at the runner code, we don't need to proxy the individual `v{1,5}` calls, but just `verbose`.  In the end the `v` methods are just shorthand to help us internally, so for reporting purposes, it makes more sense just to look at `verbose`.  A method name could be reconstructed from the `caplevel` as well.

I shuffled some code slightly, and add decorators to handle the short circuiting for `debug` and `verbose`.

Technically speaking, `system_warning` and `deprecated` both have a short circuit in them, but it's far less common that they actually get short circuited, since the defaults are `True`, so I haven't done anything with them.

File: test/integration/targets/ansible_log/runme.sh [hunk 2]
```diff
@@ -2,11 +2,16 @@
...
 ansible-playbook logit.yml
 [ ! -f "${ALOG}" ]
 
 ANSIBLE_LOG_PATH=${ALOG} ansible-playbook logit.yml
 [ -f "${ALOG}" ]
 grep -q 'ping' "${ALOG}"
+
+rm "${ALOG}"
+# inline grep should fail if EXEC was present
+ANSIBLE_LOG_PATH=${ALOG} ANSIBLE_LOG_VERBOSITY=3 ansible-playbook -v logit.yml | tee /dev/stderr | (! grep EXEC)
```
Comment (webknjaz) [inline test/integration/targets/ansible_log/runme.sh]: This is failing under macOS 13.2: https://dev.azure.com/ansible/ansible/_build/results?buildId=103411&view=logs&j=0234790f-d968-59b7-e00c-d64f03a3ea0a&t=050bde6e-6316-5116-4a50-69d2b3fe18ab&l=1806

**Review by nitzmahone — APPROVED** [review]
Yep, that oughtta cut down on wasted extraneous verbosity noise- nice to see a case where `ParamSpec` actually does the needful- I think I've only had one other case where it could actually do what I needed without being on 3.12 :laughing: .

Until we've got a threaded local manager and can just roll this stuff with autoproxy, this WFM!

**Pull request merged**